test/shaping/meson.build

```diff
@@ -0,0 +1,170 @@
+in_house_tests = [
+  ['aat-trak.tests'],
+  ['aat-morx.tests'],
+  ['arabic-fallback-shaping.tests'],
+  ['arabic-feature-order.tests'],
+  ['arabic-like-joining.tests'],
+  ['arabic-mark-attach.tests'],
+  ['arabic-mark-order.tests'],
+  ['arabic-stch.tests'],
+  ['automatic-fractions.tests'],
+  ['cluster.tests'],
+  ['collections.tests'],
+  ['color-fonts.tests'],
+  ['context-matching.tests'],
+  ['cursive-positioning.tests'],
+  ['default-ignorables.tests'],
+  ['emoji.tests'],
+  ['fallback-positioning.tests'],
+  ['hangul-jamo.tests'],
+  ['hyphens.tests'],
+  ['indic-consonant-with-stacker.tests'],
+  ['indic-decompose.tests'],
+  ['indic-init.tests'],
+  ['indic-joiner-candrabindu.tests'],
+  ['indic-joiners.tests'],
+  ['indic-old-spec.tests'],
+  ['indic-pref-blocking.tests'],
+  ['indic-script-extensions.tests'],
+  ['indic-special-cases.tests'],
+  ['indic-syllable.tests'],
+  ['indic-vowel-letter-spoofing.tests'],
+  ['khmer-mark-order.tests'],
+  ['khmer-misc.tests'],
+  ['language-tags.tests'],
+  ['ligature-id.tests'],
+  ['mark-attachment.tests'],
+  ['mark-filtering-sets.tests'],
+  ['mongolian-variation-selector.tests'],
+  ['myanmar-syllable.tests'],
+  ['myanmar-zawgyi.tests'],
+  ['none-directional.tests'],
+  ['rand.tests'],
+  ['spaces.tests'],
+  ['simple.tests'],
+  ['sinhala.tests'],
+  ['tibetan-contractions-1.tests'],
+  ['tibetan-contractions-2.tests'],
+  ['tibetan-vowels.tests'],
+  ['use.tests'],
+  ['use-indic3.tests'],
+  ['use-marchen.tests'],
+  ['use-syllable.tests'],
+  ['variations-rvrn.tests'],
+  ['vertical.tests'],
+  ['zero-width-marks.tests'],
+]
```

Comment:
This is missing new tests that were recently introduced in master, like `macos.tests`. Didn’t check the whole list.

Reply:
Went through the list, found two missing tests (kern-format2 + macos).